pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_previewing-modules-in-pantheon.adoc

```diff
@@ -0,0 +1,22 @@
+include::../attributes.adoc[]
+[id="previewing-modules-in-pantheon_{context}"]
+= Previewing modules in {ProductShortName} as logged-in user
```

Comment:
I'm not sure we need to include 'as logged-in user' in the title here - you've called that out nice and clearly in the text itself, and the general task we're attempting to perform is previewing a module. Being logged in is a detail of that, and if we included it in the title, readers might think there are other procedures for different variations.

Reply:
Indeed, changed!